drivers/google/destroy.go

```diff
@@ -13,7 +13,11 @@ import (
 )
 
 func (p *provider) Destroy(ctx context.Context, instance *autoscaler.Instance) error {
-	op, err := p.service.Instances.Delete(p.project, p.zone, instance.ID).Context(ctx).Do()
+	zone := p.zone
+	if p.region != "" {
+		zone = p.region	
+	}
```

Comment:
@TheJokersThief I made a minor change to preserve the existing behavior.  The google docs did not mention that region could be used, and I don't have enough experience with Google Cloud to confidently merge the pull request without spinning up a full environment and running some tests (which would be time consuming).  So in the interest of getting this merged quickly, I figured maybe we could implement the above logic as a compromise? Look ok to you?

Reply:
also does `waitZoneOperation` work with a region?

Reply:
also just want to confirm that if we pass `region` to the `Delete` function, will this have any impact on the subsequent `waitZoneOperation`?  Sorry, I'm not too familiar with gcp.